drcclpvmpc_main:
  ros__parameters:
    dt_: 0.05
    horizon_: 6
    track_width_: 8.0
    use_drcc: true
    control_type: "pwm"  # 'pwm'|'acc'
    # Below are parameters for the specific vehicle model
    lf: 1.3
    lr: 1.6
    mass: 1696.0
    Iz: 4036.0
    g: 9.81
    wheel_base: 1.45
    # Below are parameters for the vehicle MF tire model
    Bf: 14.3423
    Cf: 1.2712
    Df: 4.0620

    Br: 36.9947
    Cr: 4.7291
    Dr: 9.0861

    Caf: 74.06
    Car: 1589.63
    # Below are parameters for the drivetrain model
    Cm1: 6725.5478
    Cm2: 25.9941
    Cr0: 187.1197
    Cd: 1.4338

    # Below are parameters for the control limits
    max_steer: 25  # [deg]
    min_steer: -25 # [deg]

    max_vy: 3.0     # [m/s]
    min_vy: -6.0    # [m/s]

    max_vx: 8.0    # [m/s]
    min_vx: 0.0     # [m/s]

    max_acc: 3.0     # [m/s^2]
    min_acc: -6.0    # [m/s^2]

    max_omega: 30  # [rad/s]
    min_omega: -30 # [rad/s]

    # Below are parameters for the global path service name
    service_name: "get_path_from_txt"
    goal_topic: "/goal_pose"

    # Below defines the safety distance to the obstacle: safe_multiplier * max_vx
    safe_multiplier: 2.0

